src/ray/rpc/rpc_chaos.h

```diff
@@ -28,9 +28,9 @@ enum class RpcFailure {
   Response,
 };
 
-RpcFailure get_rpc_failure(const std::string &name);
+RpcFailure GetRpcFailure(const std::string &name);
 
-void init();
+void Init();
```

Comment:
Could you please add a documentation for the function?

Reply:
For example, what does it initialization? Should we call it one time or multiple times? How to reset?